docs/target-cluster.md

```diff
@@ -3,6 +3,39 @@
 Follow this guide to create a service account and generate a kubeconfig file from a [target cluster](/glossary#target-cluster).
 These are the only steps required to be performed in the target cluster.
 
+A setup [script](https://zora.undistro.io/targetcluster.sh) is also avaiable for this, which can be executed by any POSIX compliant shell:
+
+```shel
+curl -qL https://zora.undistro.io/targetcluster.sh | sh
+```
```

Comment:
We need to copy this script to the bucket like everything else that is hosted in `zora.undistro.io`. We already have automation for this.

https://github.com/undistro/zora/blob/main/.github/workflows/docs.yaml#L25
Here we need to add
`          cp hack/scripts/target_cluster.sh docs/targetcluster.sh`

A similar change should be made to the `preview-docs` target in the Makefile.

Reply:
If the goal is to have it within the `docs` dir, I can generated the file there instead.

Reply:
Are there plans to remove the docs from Zora's repo? That'd impact such approach.

Reply:
> Are there plans to remove the docs from Zora's repo? 
Not for now.

> If the goal is to have it within the docs dir, I can generated the file there instead.
Ok, it's a good option since `docs` dir is copied to bucket. Good.

Reply:
Done